kolibri_explore_plugin/assets/src/modules/topicsRoot/handlers.js

```diff
@@ -172,6 +173,21 @@ export function decideWelcome(store) {
   );
 }
 
+export function decidePackSelection(store) {
+  store.commit('CORE_SET_PAGE_LOADING', true);
+
+  if (currentLanguage.startsWith('es')) {
```

Comment:
I think this should use `languageIdToCode` rather than this type of matching since it would inadvertently match a language code like `esk` (even though there isn't one).

Reply:
Ouch! Very good point @dbnicholson

Reply:
> I think this should use `languageIdToCode` rather than this type of matching since it would inadvertently match a language code like `esk` (even though there isn't one).

I have changed the conditional to use `languageIdToCode` and rebased with master.